code/code/disc/disc_wrath.cc

```diff
@@ -13,159 +13,155 @@
 #include "obj_magic_item.h"
 #include "weather.h"
 
-int plagueOfLocusts(TBeing *caster, TBeing *victim, int level, short bKnown)
-{
-  TMonster *locusts;
-  affectedData aff; 
-  int rc, swarm;
-  followData *k, *n;
-
+int plagueOfLocusts(TBeing *caster, TBeing *victim, int level, short bKnown) {
   if (!caster->outside()) {
-    act("Nothing seems to happen.", FALSE, caster, NULL, NULL, TO_ROOM);
+    act("Nothing seems to happen.", false, caster, nullptr, nullptr, TO_ROOM);
     caster->sendTo("You need to be outside to summon locusts!\n\r");
     return SPELL_FAIL;
   }
+
   if (Weather::getWeather(*caster->roomp) == Weather::RAINY) {
-    act("Nothing seems to happen.", FALSE, caster, NULL, NULL, TO_ROOM);
+    act("Nothing seems to happen.", false, caster, nullptr, nullptr, TO_ROOM);
     caster->sendTo("How do you expect to summon locusts in the rain!\n\r");
     return SPELL_FAIL;
   }
-  if (level < 26) 
-    swarm = Mob::LOCUSTS25;
-  else if (level < 31) 
-    swarm = Mob::LOCUSTS30;
-  else if (level < 36) 
-    swarm = Mob::LOCUSTS35;
-  else if (level < 41) 
-    swarm = Mob::LOCUSTS40;
-  else 
-    swarm = Mob::LOCUSTS50;
-
-  if (!(locusts = read_mobile(swarm, VIRTUAL))) {
-    vlogf(LOG_BUG, "Spell PLAGUE_LOCUSTS unable to load mob...");
-    caster->sendTo("Unable to create the locusts, please report this.\n\r");
-    act("Nothing seems to happen.", FALSE, caster, NULL, NULL, TO_ROOM);
-    return SPELL_FAIL;
-  }
 
-  if ((caster->followers) && (level < GOD_LEVEL1))  {
-    for (k = caster->followers; k; k = n) {
-      n = k->next;
-      if (isname("locust", k->follower->name)) {
-        act("Nothing seems to happen.", FALSE, caster, NULL, NULL, TO_ROOM);
-        caster->sendTo("You would have to be a god to summon another locust swarm!\n\r");
-        delete locusts;
-        locusts=NULL;
+  if ((caster->followers) && (level < GOD_LEVEL1))  {    
+    for (auto currentFollower = caster->followers; currentFollower; currentFollower = currentFollower->next) {      
+      if (isname("locust", currentFollower->follower->name)) {
+        act("Nothing seems to happen.", false, caster, nullptr, nullptr, TO_ROOM);
+        caster->sendTo("You would have to be a god to summon another locust swarm!\n\r");        
         return SPELL_FAIL;
-      }
+      }      
```

Comment:
Let's remove those trailing spaces

Reply:
I'm going to run the code through VS Code's C++ auto-formatter that I've been afraid to use before now because it would generate so many file changes. Get rid of all the extraneous whitespace and other hard-to-see formatting things at once.

Let me know if it does anything in a way you'd prefer done differently - I can customize pretty much everything about it.

Reply:
Fixed in https://github.com/sneezymud/sneezymud/pull/353/commits/d4b339ee3f5aa092aff3194c561eb00b2806589c.

Glad you caught that, there was a lot more than just that one spot.